brightray/browser/browser_context.cc

```diff
@@ -214,18 +192,24 @@ BrowserContext::CreateRequestContextForStoragePartition(
     bool in_memory,
     content::ProtocolHandlerMap* protocol_handlers,
     content::URLRequestInterceptorScopedVector request_interceptors) {
+  NOTREACHED();
   return nullptr;
 }
 
 net::URLRequestContextGetter* BrowserContext::CreateMediaRequestContext() {
-  return url_request_getter_.get();
+  return io_handle_->GetMainRequestContextGetter().get();
 }
 
 net::URLRequestContextGetter*
 BrowserContext::CreateMediaRequestContextForStoragePartition(
     const base::FilePath& partition_path,
     bool in_memory) {
+  NOTREACHED();
   return nullptr;
 }
 
+std::string BrowserContext::GetUserAgent() const {
+  return std::string();
```

Comment:
Is this intentional? 

If so, it it a placeholder or something required because reasons, e.g. filling in a pure virtual function?

Either way, could you add a comment explaining intent so that future code archaeologists will understand :smile:

Reply:
You are right, this is not needed. The actual implementation comes from `AtomBrowserContext`, have removed this definition. Thanks!